roles-and-groups.md

```diff
@@ -0,0 +1,152 @@
+Like with the nomination of a Committer, the vote of the maintainers should be held in a GitHub PR (by doing a change to the `config.yaml` file).
+
+### [Maintainers](#maintainers)
+
+The Maintainers of the project are the people that are responsible for the project.
+
+A Maintainer should drive the project's roadmap/vision and take care of topics like community meetings,
+the general structure of the project (fits our best practices, is secure, ...), and so on.
+
+In addition, the Maintainers should report project updates to the Hiero **Technical Steering Committee** (TSC).
+A Committer of a project can be nominated to become a Maintainer:
+
+- **Define your project's needs**: A repository should have 1 to 3 maintainers to ensure a project's sustainability and resilience. It is not necesarily a goal for the committer to become a mintainer unless the project's needs require it to have more maintainers.
+- **Nominate a maintainer**: To promote a candidate, a vote should be held in a GitHub PR (by doing a change to the config.yaml file) and by making mention of the basis reached for nominating the Maintainer.
+- **Approve the nomination**: All Maintainers should vote on that nomination and the vote passes when reaching majority of votes.
+- **Make adjustments**: A maintainer is considered inactive if there has been no activity for six months and there has been no response when trying to contact them.
+
+A maintainer can lose the role if:
+
+- Breaks Code of Conduct.
+- Are inactive for 6 months and does not respond when reached out to.
+- Keeps making bad decisions (their contributions are actively affecting the project's performance).
+
+More on maintainer guidelines can be found on the [**LFDT TAC site**](https://lf-decentralized-trust.github.io/governance/governing-documents/MAINTAINERS-file.html).
+
+### [Voting](#voting)
+
+When it comes to electing and voting for new committers or new maintainers, the project should consider the following:
+
+For candidates working on getting promoted to Committers or Maintainers:
+- The candidate needs to show participation in the project by contributing with code, issues, discussions, reviews, etc.
+- These contributions need to show proper understanding of the project and quality of the work.
+- 3 to 6 months worth of contributions is considered a good minimum for candidates to be selected for a role promotion.
+- A Committer or Maintainer will be considered inactive if they shown no activity for 6 months at least, in such case, they could be selected for removal.
+- A Committer can maintain their privileges independently from their job changes.
+
+For Maintainers and Committers voting on new candidates:
+- When a candidate is selected for becoming a Committer or a Maintainer, this proposal needs to be made via PR request against the config.yaml in the governance repo.
+- An example of a promotion can be found in the following [PR](https://github.com/hiero-ledger/governance/pull/208).
```

Comment:
https://github.com/hiero-ledger/governance/pull/176 should be used as provides a clear timeline and means for a full quorum vote (including for, against, abstain, and not present/no vote cast).

```suggestion
- An example of a promotion can be found in the following [PR](https://github.com/hiero-ledger/governance/pull/176).
```